requirements/base.txt

```diff
@@ -10,3 +10,5 @@ git+https://github.com/Doca/django-ajax.git#egg=djangoajax
 python3-memcached==1.51
 
 ipython==8.2.0
+
+sentry-sdk==1.5.8
```

Comment:
Should it be in base.txt or production, what's the point of having it locally or in unit tests?

Reply:
Good point.
Though it's now in two files (production.txt and postgres.txt), but we should clean this mess up nevertheless.
Fixed.